### DISCUSSION TITLE
TP4 - ejercicio 6 - Intercambio de caracters

### DISCUSSION THREAD

**Discussion by luciano-repetti** [INGCOM-UNRN-P1/cursada-2023#136] [Preguntas y respuestas] [answered]
Buenas! En el enunciado del ejercicio 6 dice "Intercambiar dos caracteres en una misma cadena segura.".

Prototype: ``` int intercambia(char cadena[], int capacidad_cadena, int origen, int destino); ```

```int origen``` e ```int destino``` deben ser las posiciones del array para intercambiar los valores o deben ser caracters que se cambian siempre y cuando se encuentre los caracters pasados por argumento.

Para mi es la posición que se debe cambiar, pero esto no necesita un lazo, pero entonces para que el argumento ```int capacidad_cadena```

**Comment by martinvilu** [comment]
Son dos posiciones.

Sin `capacidad`, no hay cadena segura.

**Reply by luciano-repetti** [reply]
Osea que no es necesario un bucle, sería algo asi:

<Código del repositorio>

**Reply by martinvilu** [reply, marked as answer]
Sí, y usá la función de intercambio ya que estamos.
